src/model/MazeSizeException.java

```diff
@@ -0,0 +1,7 @@
+package model;
+
+public class MazeSizeException extends Exception {
```

Comment:
It's alright, but looks a bit off as this is the only exception class defined in this project

Reply:
Right, it was kind of the start of custom made exceptions  🙅‍♂️